src/middlewares/authenticationMiddleware.ts

```diff
@@ -0,0 +1,46 @@
+import jwt from 'jsonwebtoken'
+import { Response, Request, NextFunction } from 'express'
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace Express {
+    // eslint-disable-next-line no-shadow
+    interface Request {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      userDetails: any
+    }
+  }
+}
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
```suggestion
declare global {
  namespace Express {
    interface Request {
      userDetails: any
    }
  }
}

```
Lets not disable eslint